### ISSUE TITLE
Error during import of seed mechanism from previous simulation

### ISSUE THREAD

**Issue by cgrambow** [ReactionMechanismGenerator/RMG-Py#1204]
I'm trying to use the core of a previous RMG simulation as the seed mechanism for a new simulation and the edge from the previous simulation as a reaction library. The seed and edge from the previous simulation were saved by setting `generateSeedEachIteration=True`. I then import them in the new simulation using `seedMechanisms = [<previous core>]` and `reactionLibraries = [<previous edge>]` where I added `<previous core>` and `<previous edge>` to the database.

RMG starts off by loading files:
```
Loading kinetics library decyl_cyclohexane_021 from /data1/cgrambow/Code/RMG-database/input/kinetics/libraries/decyl_cyclohexane_021/reactions.py...
Loading kinetics library decyl_cyclohexane_021_edge from /data1/cgrambow/Code/RMG-database/input/kinetics/libraries/decyl_cyclohexane_021_edge/reactions.py...
Loading frequencies group database from /data1/cgrambow/Code/RMG-database/input/statmech/groups...
Loading Platts additivity group database from /data1/cgrambow/Code/RMG-database/input/solvation/groups...
Adding rate rules from training set in kinetics families...
Filling in rate rules in kinetics families by averaging...
Enabling diffusion-limited kinetics...
Setting solvent data for decylcyclohexane
Adding seed mechanism decyl_cyclohexane_021 to model core...
```

It then prints a bunch of messages like this:
```
H_Abstraction
 [C/H/Cs3;O2b]
R_Recombination
 [O2_birad;C_rad/Cs3]
R_Recombination
 [O_rad/NonDe;C_rad/Cs3]
[...]
intra_H_migration
 [R2H_S;C_rad_out_H/NonDeC;Cs_H_out_H/(NonDeC/Cs)]
H_Abstraction
 [C/H2/NonDeC;C_rad/H/NonDeC]
Ea raised from -36.0 to -36.0 kJ/mol.
 [ROOH_sec;C_rad/H/NonDeC]
[...]
H_Abstraction
 [ROOH_sec;OOC]
R_Recombination
 [O_rad/NonDe;C_rad/H/NonDeC]
```
where I omitted most of the messages as indicated by `[...]`. Finally, I get this error:
```
 [R3OO_SS;C_rad/H/NonDeC_intra;OOH]
Traceback (most recent call last):
  File "/data1/cgrambow/Code/RMG-Py/rmg.py", line 173, in <module>
    main()
  File "/data1/cgrambow/Code/RMG-Py/rmg.py", line 167, in main
    rmg.execute(**kwargs)
  File "/data1/cgrambow/Code/RMG-Py/rmgpy/rmg/main.py", line 531, in execute
    self.initialize(**kwargs)
  File "/data1/cgrambow/Code/RMG-Py/rmgpy/rmg/main.py", line 420, in initialize
    self.reactionModel.addSeedMechanismToCore(seedMechanism, react=False)
  File "/data1/cgrambow/Code/RMG-Py/rmgpy/rmg/model.py", line 1372, in addSeedMechanismToCore
    r, isNew = self.makeNewReaction(rxn) # updates self.newSpeciesList and self.newReactionlist
  File "/data1/cgrambow/Code/RMG-Py/rmgpy/rmg/model.py", line 500, in makeNewReaction
    found, rxn = self.checkForExistingReaction(forward)
  File "/data1/cgrambow/Code/RMG-Py/rmgpy/rmg/model.py", line 398, in checkForExistingReaction
    familyObj = getFamilyLibraryObject(rxn.family)
  File "/data1/cgrambow/Code/RMG-Py/rmgpy/rmg/model.py", line 1880, in getFamilyLibraryObject
    raise Exception('Could not retrieve the family/library: {}'.format(label))
Exception: Could not retrieve the family/library: 
```

Somehow RMG can't load a family/library even though the previous model was generated using the same settings.

**Comment by mjohnson541** [comment]
This particular issue is fixed by #1200.

**Comment by mjohnson541** [comment]
However, there's another issue I've run into loading larger seeds that I haven't fixed yet.
